src/ui/simulator/application/main/logs.cpp

```diff
@@ -206,9 +206,9 @@ void ApplWnd::connectLogCallback()
     logs.callback.connect(this, &ApplWnd::onLogMessageDeferred);
 }
 
-void ApplWnd::onLogMessageDeferred(int level, const String& message)
+void ApplWnd::onLogMessageDeferred(int level, const std::string& message)
 {
-    if (not message.empty() and message.first() != '[')
+    if (not message.empty())
```

Comment:
Behavior changes

Reply:
didn't bother bc it's logs in the GUI

done